nodejs/week1/avg.js

```diff
@@ -0,0 +1,17 @@
+function calcAvg (numbers) {
+    let sum = 0;
+    if(numbers.length === 0 || numbers.some(isNaN)) {
+        return `Please enter valid numbers.`
+    }else {
+        numbers.forEach((num) => {
+            sum += num;
+        });
```

Comment:
Nice use of the `forEach` function. As a suggestion, you could have also used the `reduce()` function. 😉